ge_query.py

```diff
@@ -0,0 +1,29 @@
+import requests
+import pdb
+
+def grand_exchange_query(query_type=None,itemID = None):
+    base_url = "http://services.runescape.com/m=itemdb_oldschool"
+    type_dict = {
+            'detailed' : '/api/catalogue/detail.json?item=itemID',
+            'time_data': '/api/graph/itemID.json',
+            }
+    
+    query_url = type_dict[query_type]
+    if 'itemID' in query_url and itemID is None:
+        raise AttributeError('When `query_type` = ' + query_type + ' `itemID` must be defined')
```

Comment:
Why not use a fstring here? `f'When `query_type` = {query_type} `itemID` must be defined'`